app/controllers/api/v1/accounts/conversations_controller.rb

```diff
@@ -36,6 +36,10 @@ def create
     end
   end
 
+  def update
+    @conversation.update!(permitted_update_params)
```

Comment:
Would this allow updating a conversation with different sla_policies? We should restrict updating a `conversation` with another policy if SLA is already set.

Reply:
We can handle this validation in the model 👍